bluebutton-sample-config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Just a reminder that these files will be inside the SDK and the 3rd party app wont have access to them (just think when you install a library via pip, you don't generally copy files out of it). So I'm not sure here is where we want our sample files, probably just show how to create them in the read me.

Reply:
@nbragdon YES! This is a good change. I'll remove these files and include them in the README.md. The SDK user can copy the example configs from there.  TY!

Reply:
@nbragdon Removed the sample config file.